scripts/compare_benchmarks.py

```diff
@@ -12,10 +14,16 @@
 min_runtime_ns = 59 * 1000 * 1000 * 1000
 
 def format_diff(diff):
-    if diff >= 0:
-        return colored('+' + "{0:.0%}".format(diff), 'green')
-    else:
+    if diff < 0:
         return colored("{0:.0%}".format(diff), 'red')
+    else:
+        return colored("+{0:.0%}".format(diff), 'green')
+
+def format_change(change):
+    if change < 0:
+        return colored("▼ {:.4f}".format(abs(change)), 'red')
+    else:
+        return colored("▲ {:.4f}".format(abs(change)), 'green')
```

Comment:
grey for `< 1.05`?

Reply:
Rather standard white for "non significant" results? Grey might look strange when everything else is white?

Reply:
Sure

Reply:
Careful though. I'm talking about irrelevant changes (just doesn't change a lot), not insignificant changes (p-value)

Reply:
That’s what I meant as well.